handpose/src/index.ts

```diff
@@ -67,15 +63,21 @@ interface AnnotatedPrediction extends Prediction {
  * Defaults to 0.3.
  * - `scoreThreshold` A threshold for deciding when to remove boxes based
  * on score in non-maximum suppression. Defaults to 0.75.
+ * - `handdetectModelURL` Specify URL of handdetect model. Defaults to tfhub.dev.
+ * - `handposeModelURL` Specify URL of handdetect model. Defaults to tfhub.dev.
+ * - `anchorsURL` Specify URL of handdetect model's anchors file. Defaults to tfhub.dev.
  */
 export async function load({
   maxContinuousChecks = Infinity,
   detectionConfidence = 0.8,
   iouThreshold = 0.3,
-  scoreThreshold = 0.5
+  scoreThreshold = 0.5,
+  handdetectModelURL = 'https://tfhub.dev/mediapipe/tfjs-model/handdetector/1/default/1',
+  handposeModelURL = 'https://tfhub.dev/mediapipe/tfjs-model/handskeleton/1/default/1',
+  anchorsURL = 'https://tfhub.dev/mediapipe/tfjs-model/handskeleton/1/default/1/anchors.json?tfjs-format=file'
```

Comment:
can we call these `handdetectorModelURL` and `handskeletonModelURL`?

Reply:
I think URL is a broad concept and `path` is a type of URL.
In `loadGraphMode` function,  it is called `modelURL` and supports google storage URL or tfhub path.